### ISSUE TITLE
File too large to push

### ISSUE THREAD

**Issue by gordonty** [Montana-Media-Arts/341-work#35]
My video for my hosted video is too large to push. I have tried git large file storage and zipping the video but it still won't push through because the video file is still too large. Anyone have any ideas at all.

**Comment by Jessie-Devine** [comment]
Try lowering the video quality and re-exporting it. You could also try changing the video format. I don't know what it's in now but if you export it in a lossy format it'll save you some space.

**Comment by michaelmusick** [comment]
Yeah, you have to get the size down. Ideally, the video size should be less than 200MB or so. 
Each git repository is limited to a total size of 2GB. Soooooo…… depending on the duration of the video, you will either need to cut it down, or compress the heck out of it (or maybe both????).

Storage space is one of the reasons why self-hosting videos can be hard.

**Comment by gordonty** [comment]
How do I compress it I've tried everything. I did at one point get it from a .MOV to .MP4 but it still wouldn't push through

**Comment by MediaArts-Montana** [comment]
I am assuming you are on a mac since you have not provided info otherwise.

One way is to open the video in Quicktime. Then under File, select Export>ipod,iphone,etc

[cid:C8210040-FDAD-4362-B8DF-EDC604E6EDD5@CasaRincon]

On mac or windows, a similar type of option is possible with [VLC](http://www.videolan.org/vlc/index.html)

Select the export/save as option under File for VLC, and then the appropriate settings.

**Comment by hyeink** [comment]
I made the file size down, but it still doesn't work.
